workspace_profile/workspace_profile_steampipe.go

```diff
@@ -9,33 +9,48 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/turbot/pipe-fittings/constants"
 	"github.com/turbot/pipe-fittings/cty_helpers"
+	"github.com/turbot/pipe-fittings/error_helpers"
 	"github.com/turbot/pipe-fittings/hclhelpers"
 	"github.com/turbot/pipe-fittings/options"
 	"github.com/zclconf/go-cty/cty"
 )
 
 type SteampipeWorkspaceProfile struct {
-	ProfileName       string                     `hcl:"name,label" cty:"name"`
-	PipesHost         *string                    `hcl:"pipes_host,optional" cty:"pipes_host"`
-	PipesToken        *string                    `hcl:"pipes_token,optional" cty:"pipes_token"`
-	InstallDir        *string                    `hcl:"install_dir,optional" cty:"install_dir"`
-	QueryTimeout      *int                       `hcl:"query_timeout,optional" cty:"query_timeout"`
-	SnapshotLocation  *string                    `hcl:"snapshot_location,optional" cty:"snapshot_location"`
-	WorkspaceDatabase *string                    `hcl:"workspace_database,optional" cty:"workspace_database"`
-	SearchPath        *string                    `hcl:"search_path" cty:"search_path"`
-	SearchPathPrefix  *string                    `hcl:"search_path_prefix" cty:"search_path_prefix"`
-	MaxParallel       *int                       `hcl:"max_parallel" cty:"max-parallel"`
-	Input             *bool                      `hcl:"input" cty:"input"`
-	Progress          *bool                      `hcl:"progress" cty:"progress"`
-	Theme             *string                    `hcl:"theme" cty:"theme"`
-	Cache             *bool                      `hcl:"cache" cty:"cache"`
-	CacheTTL          *int                       `hcl:"cache_ttl" cty:"cache_ttl"`
-	Base              *SteampipeWorkspaceProfile `hcl:"base"`
+	ProfileName string `hcl:"name,label" cty:"name"`
```

Comment:
move all deprecated options to the end with a single comment:
// no longer supported, exists here only to avoid errors